pkg/java/v1/api.go

```diff
@@ -17,34 +17,47 @@
 package v1
 
 import (
+
+	"errors"
 	"fmt"
 
 	"github.com/spf13/pflag"
 	"sigs.k8s.io/kubebuilder/v3/pkg/config"
 	"sigs.k8s.io/kubebuilder/v3/pkg/machinery"
+	"sigs.k8s.io/kubebuilder/v3/pkg/model/resource"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
+	pluginutil "sigs.k8s.io/kubebuilder/v3/pkg/plugin/util"
 
-	"github.com/java-operator-sdk/kubebuilder-plugin/pkg/internal/kubebuilder/cmdutil"
 	"github.com/java-operator-sdk/kubebuilder-plugin/pkg/java/v1/scaffolds"
 )
 
+type createAPIOptions struct {
+	CRDVersion         string
+	DoRole, DoPlaybook bool
```

Comment:
These are ansible specific. Remove them.
```suggestion
```

Reply:
Modified according to suggestion.